card.ml

```diff
@@ -0,0 +1,60 @@
+open Yojson.Basic.Util
+
+(**************************************************************************)
```

Comment:
Why have these comment lines? You could omit them

Reply:
They just helped me group sections of code that relate to each other